src/box/txn.c

```diff
@@ -909,6 +920,10 @@ txn_journal_entry_new(struct txn *txn)
 		if (stmt->row == NULL)
 			continue;
 
+		/* A meta-temporary's space metadata change. */
+		if (stmt->is_temporary)
+			continue;
```

Comment:
Will it work, if we drop `stmt->is_temporary` and just nullify `stmt->row` for those meta-statements?

Reply:
Do I understand correctly, that memory pointed to by `stmt->row` is managed by `txn->region` doesn't need manual freeing?

Reply:
> Do I understand correctly, that memory pointed to by stmt->row is managed by txn->region and doesn't need manual freeing?

Yes. Or might be pointing at something on stack somewhere. Anyway, no freeing.

Reply:
done